src/Slack/Provider.php

```diff
@@ -102,7 +102,12 @@ protected function getUserByToken($token)
     {
         try {
             $response = $this->getHttpClient()->get(
-                'https://slack.com/api/users.identity?token='.$token
+                'https://slack.com/api/users.identity',
+                [
+                    'headers' => [
+                        'Authorization' => 'Bearer '.$token,
```

Comment:
Link to the docs please for this change? (but good to not pass it in the query :))